src/transformStateWithClones.js

```diff
@@ -83,15 +85,14 @@ function transformStateWithClones(state, transforms) {
         break;
 
       case 'removeProperties':
-        for (let j = 0; j < transform.properties.length; j++) {
-          delete currentObject[transform.properties[j]];
+        for (const property in transform.properties) {
```

Comment:
use for..of instead of for..in, since transform.properties is an array

Reply:
transform.properties is an object, so I use for ... in loop